src/infrastructure/figma/transformers/figma-file-transformer.ts

```diff
@@ -0,0 +1,38 @@
+import {
+	buildDesignUrl,
+	buildInspectUrl,
+	buildLiveEmbedUrl,
+	getUpdateSequenceNumber,
+} from './utils';
+
+import type { AtlassianDesign } from '../../../domain/entities';
+import {
+	AtlassianDesignStatus,
+	AtlassianDesignType,
+	FigmaDesignIdentifier,
+} from '../../../domain/entities';
+import type { FileResponse } from '../figma-client';
+
+type TransformFileToAtlassianDesignParams = {
+	readonly fileKey: string;
+	readonly fileResponse: FileResponse;
+};
+
+export const transformFileToAtlassianDesign = ({
```

Comment:
**note:** This code has been moved from `infrastructure/figma/figma-transformer.ts` with no changes.